README.md

```diff
@@ -1,9 +1,266 @@
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
```

Comment:
Great job on the customization of the `README`. Please remove this code as you don't have your own logo and so it should be blank space.
![Screenshot from 2023-06-01 20-03-09](https://github.com/thasipablo/Hello-Microverse/assets/101523582/0c957ef4-799d-46e4-ad1a-6aaa1df60255)
Kindly, fix it.